### DISCUSSION TITLE
Working with files of more than 30 seconds

### DISCUSSION THREAD

**Discussion by gregdrizz** [openai/whisper#354] [Q&A] [answered]
What would be the best way to work with audio files that are longer than 30 seconds, I mean how'd you do it if i want to transcribe a whole 2 minute recording?

**Comment by jianfch** [comment, marked as answer]
Unless you're using the lower-level methods, `transcribe()` and cli already perform long-form transcription (transcribe audio longer than 30sec).

**Reply by Pompey21** [reply]
Hi, I am using the following

model = whisper.load_model('base')
result = model.transcribe(path_to_wav_file)

but I only obtain text from first 10 seconds

**Reply by Majdoddin** [reply]
Is there a speaker overlap? #434
